examples/tutorial_builder/README.md

```diff
@@ -22,6 +22,8 @@
 
 You can create an application with several arguments using usage strings.
 
+Note: This example uses the `command!()` macro. This requires the `cargo` feature flag.
+
```

Comment:
Trying to decide the route we should go.  The intention here is to not be pedagogical (like the later ones) but to provide a sample of what can be done.  I feel like the current approach in master accomplishes that though I can see why others might not agree.

If we move forward with this, we should keep the focus on the link to the code (which notes the needed feature) and the help output.  I'd recommend italicizing this note and putting it after the `console` output.